lib/streamlit/server/routes.py

```diff
@@ -54,6 +54,11 @@ def set_extra_headers(self, path):
             self.set_header("Cache-Control", "public")
 
 
+class AssetsFileHandler(tornado.web.StaticFileHandler):
+    def set_default_headers(self):
```

Comment:
Let's add a comment here explaining why it's OK that we disable CORS protection for this endpoint

Reply:
Done!